akka-remote/src/main/mima-filters/2.5.25.backwards.excludes

```diff
@@ -0,0 +1,7 @@
+# #27411 changes not caught by previous mima versions
+## ActorRef became an OptionVal
+ProblemFilters.exclude[IncompatibleSignatureProblem]("akka.remote.transport.AkkaPduCodec#Message.unapply")
+## StopSignal became OptionVal[StopSignal]
+ProblemFilters.exclude[IncompatibleSignatureProblem]("akka.remote.artery.Association#OutboundStreamMatValues.unapply")
+## ActorRef became OptionVal[ActorRef]
+ProblemFilters.exclude[IncompatibleSignatureProblem]("akka.remote.EndpointManager#Send.unapply")
```

Comment:
These are unexpected: I wonder if perhaps mima gets confused by the use of `OptionVal`. Let's see if they pop back up later, if so we might need further action.

Reply:
It's internals anyway, so we don't care?

Reply:
Sure - I'm just concerned this might be a mima problem and we'll have to add the filters again for each release. But perhaps let's cross that bridge when we get there.

Reply:
Seems the signatures for types with value types in arguments was wrong before 2.12.9: https://github.com/scala/scala/commit/369c74329b3d5d30bd2f5d9c6d2107d3d7af125b. This was fixed for 2.12.9. So, MIMA was correct here, though the signatures are only interesting for Java consumers anyway.

Reply:
Aah cool!